lib/src/messages/customer.dart

```diff
@@ -40,3 +44,20 @@ class Customer extends Message {
   @override
   Map<String, dynamic> toJson() => _$CustomerToJson(this);
 }
+
+/// The customer’s default invoice settings.
+@JsonSerializable()
+class InvoiceSettings {
```

Comment:
This may be against the codestyle of the repo in its current state, but I propose leaving links to Stripe docs for convenience:
```suggestion
/// The customer’s default invoice settings.
/// https://docs.stripe.com/api/customers/object#customer_object-invoice_settings
@JsonSerializable()
class InvoiceSettings {
```